guide/java/topology-dependencies.md

```diff
@@ -0,0 +1,49 @@
+---
+layout: website-normal
+title: Topology, Dependencies, and Management Policies
+title_in_menu: Topology, Dependencies, and Management Policies
+---
+
+Applications written in YAML can similarly be written in Java. However, the YAML approach is 
+recommended.
+
+## Define your Application Blueprint
+
+The example below creates a three tier web service, composed of an Nginx load-balancer, 
+a cluster of Tomcat app-servers, and a MySQL database:
+
```

Comment:
Perhaps make this the java equivalent of the YAML one in /guide/start/policies.html and mention that here?